src/UI/Seller/src/app/buyers/components/buyers/buyer-edit/buyer-edit.component.ts

```diff
@@ -31,22 +35,40 @@ export class BuyerEditComponent implements OnDestroy {
   @Input()
   set orderCloudBuyer(buyer: HSBuyer) {
     if (buyer.ID) {
+      this.getBuyerData(buyer.ID)
       this.handleSelectedBuyerChange(buyer)
     } else {
       this.refreshBuyerData(this.buyerService.emptyResource)
     }
   }
+
   @Output()
   resourceDelete = new EventEmitter<any>()
 
   constructor(
     private buyerService: BuyerService,
-    private ocImpersonationService: OcImpersonationConfigService,
     private router: Router,
+    private translate: TranslateService,
     private buyerTempService: BuyerTempService,
-    private appAuthService: AppAuthService
+    private hsCatalogService: CatalogsTempService,
+    private addressService: OcAddressService
```

Comment:
Don't we want to use the javascript sdk's equivalent instead of angular? I think soon we'll want to remove this dependency so we just have one sdk to interact with (the javascript one)

Reply:
just updated